pyproject.toml

```diff
@@ -115,7 +115,14 @@ exclude = ["tests/static/**"]
 
 [tool.setuptools.package-data]
 ethereum_test_forks = ["forks/contracts/*.bin"]
+"pytest_plugins.execute" = ["eth_config/networks.yml"]
+"pytest_plugins.consume" = [
+    "simulators/hive_tests/__init__.py",
+    "simulators/hive_tests/test_via_*.py",
+    "direct/test_via_direct.py"
+]
 "cli.eest.make" = ["templates/*.j2"]
+"cli.pytest_commands" = ["pytest_ini_files/*.ini"]
```

Comment:
Love it. Would have preferred a separate PR 😂 

Closes #1885

Reply:
Could you please move `fill`'s `pytest.ini` here too, required for #1885. But let's do `pytest-framework.ini` -> `pyproject.toml` in a follow-up.